tests/test_logging.py

```diff
@@ -0,0 +1,160 @@
+import datetime
+import unittest
+
+from core_lib.helpers.logging import Logging
+
+
+class TestLogging(unittest.TestCase):
+    def test_string_logger(self):
+        string = "hello"
+
+        with self.assertLogs('TestLogging.logging_message_params', level='INFO') as cm:
+            self.logging_message_params(string)
+        self.assertEqual(cm.output, ["INFO:TestLogging.logging_message_params:log_for_test.hello"])
+
+        with self.assertLogs('TestLogging.logging_message_params', level='INFO') as cm:
+            self.logging_message_params(None)
+        self.assertEqual(cm.output, ["INFO:TestLogging.logging_message_params:log_for_test.!EparamE!"])
+
+        with self.assertLogs('TestLogging.logging_message_params', level='INFO') as cm:
+            self.logging_message_params("")
+        self.assertEqual(cm.output, ["INFO:TestLogging.logging_message_params:log_for_test.!EparamE!"])
+
+        with self.assertLogs('TestLogging.logging_message', level='INFO') as cm:
+            self.logging_message(string)
+        self.assertEqual(cm.output, ["INFO:TestLogging.logging_message:log_for_test."])
+
+        with self.assertLogs('TestLogging.logging_params', level='INFO') as cm:
+            self.logging_params(string)
+        self.assertEqual(cm.output, ["INFO:TestLogging.logging_params:.hello"])
+
+    def test_object_logger(self):
+        dat = datetime.date(2022, 1, 1)
+        obj = {'date': dat, 'tuple': ("fruit", "apple"), 'list': ["fruit", "apple"]}
+
+        with self.assertLogs('TestLogging.logging_message_params', level='INFO') as cm:
+            self.logging_message_params(obj)
+        self.assertEqual(
+            cm.output,
+            [
+                "INFO:TestLogging.logging_message_params:log_for_test.{'date': datetime.date(2022, 1, 1), 'tuple': ('fruit', 'apple'), 'list': ['fruit', 'apple']}"

```

Comment:
good test, thank you. 
here is another interesting point. 
```Python
    @Logging(log_parameters=True)
     def login(self, credentials):
         # login using credentials, 
         # {'username': 'user@abc.com',
         #   'password': 'please don't print passwords '}
````

log passwords is a security risk. if a server get hacked. they will look for sensetive data in log files. 
also what happen when we pass as a parameter a huge object. a list of 10000000 users. 

lets print the value only if it's a string or a number. else print the type.